.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
why do we need to ignore this?

Reply:
the target folder is the maven default output folder. I think it s normal to ignore maven target, just like on the step portfolio website. I followed it : )

Reply:
Got it. Thanks! :D